backend/Contracts/DTOs/RestaurantDto.cs

```diff
@@ -23,14 +23,21 @@ public RestaurantDto
             string address,
             Coords coords,
             string description = "",
-            string imageURL = DEFAULT_IMAGE_URL
+            string imageURL = ""
```

Comment:
Is it possible to do it as an optional argument and do something similar to what we do in food constructor:
```csharp
CreatedAt = createdAt ?? DateTime.Now;
```

Reply:
When the api call is made for some reason it creates an object without using the constructor. It just uses the setters and sets the values so these optional arguments are skipped. I don't know how to fix it correctly.

Reply:
And this is an optional argument because its an empty string if not given.